smart-hadoop-support/smart-hadoop/src/main/java/org/smartdata/hdfs/metric/fetcher/NamespaceFetcher.java

```diff
@@ -143,28 +181,47 @@ public void run() {
         }
         return;
       }
+      String configString = conf.get(SmartConfKeys.SMART_SERVER_IGNORE_FILE_DIRS_KEY);
```

Comment:
Line 184-190 should be moved out from 'run' as it will not change during the process, also it equals conf.getTrimmedStringCollection().